alumnium/drivers/playwright_driver.py

```diff
@@ -15,10 +16,13 @@ class PlaywrightDriver(BaseDriver):
     NOT_SELECTABLE_ERROR = "Element is not a <select> element"
     CONTEXT_WAS_DESTROYED_ERROR = "Execution context was destroyed"
 
-    with open("./scripts/waiter.js") as f:
```

Comment:
sorry-sorry!

Reply:
no worries, I was bitten by the same issue before - c7a4099